.github/workflows/deploy-production.yml

```diff
@@ -69,6 +69,10 @@ jobs:
             project: carto-un-eu
             bucket: gs://spatialextension_un/
             group: spatialextension_users_un@cartodb.com
+          - region: na-ne1
```

Comment:
I think you need to put here the full name for the region: `northamerica-northeast1`

Reply:
You are right: https://github.com/CartoDB/carto-spatial-extension/pull/194/commits/782dca4fca1f0aa35a68e4526075983bd672d041